cmd/admin.go

```diff
@@ -233,7 +234,10 @@ func runChangePassword(c *cli.Context) error {
 	if err := initDB(); err != nil {
 		return err
 	}
-
+	if !util.CheckPasswordComplexity(c.String("password")) {
+		err := errors.New("PasswordComplexity")
+		return err
```

Comment:
Why not just `return errors.New("PasswordComplexity")` and the error description is not clear.